robotbase/src/main/java/com/team2357/frc2022/commands/human/ClimbProgressionCommand.java

```diff
@@ -1,15 +1,49 @@
 package com.team2357.frc2022.commands.human;
 
+import com.team2357.frc2022.commands.climb.ClimberClimbToReachableCommandGroup;
+import com.team2357.frc2022.commands.climb.ClimberClimbToRungCommandGroup;
 import com.team2357.lib.commands.CommandLoggerBase;
 
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
 public class ClimbProgressionCommand extends CommandLoggerBase {
+    private SequentialCommandGroup climbCommands[] = {
+            new ClimberClimbToReachableCommandGroup(),
```

Comment:
We need 4 steps:

0. Raise up climber arms
1. Pull up to Mid
2. Transfer to High
3. Transfer to Traversal